openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -199,6 +200,14 @@ def clean_email(self):
         # The line below contains the only change, removing is_active=True
         self.users_cache = User.objects.filter(email__iexact=email)
 
+        # This filter allows filtering users before resetting password
```

Comment:
Please add a big fat `# EDLYCUSTOM` comment, such that we can quickly identify changes across our codebase in the future.